### ISSUE TITLE
Device does not change parameters/changes leads to same value

### ISSUE THREAD

**Issue by TutenStain** [zwave-js/zwave-js-ui#2700]
I have a couple of Everspring AC301 TRVs and since a while (cannot pinpoint when, could be months) they stopped responding to parameter changes. Example: I change `Thermostat Mode v3 [84-64-0-mode] Thermostat mode` from heat to energy heat but the setting is never updated on the device. Going by the logs it seems to attempt to change to the same value it currently has? 

The log:

`2022-10-01 15:01:00.871 INFO Z-WAVE: Node 84: value updated: 64-0-mode 1 => 1`

```
 2022-10-01T13:01:00.874Z - VALUE UPDATED 
Arg 0:
└─commandClassName: Thermostat Mode
└─commandClass: 64
└─property: mode
└─endpoint: 0
└─newValue: 1
└─prevValue: 1
└─propertyName: mode
```

The full debug info log is attached if needed.

[log.txt](https://github.com/zwave-js/zwave-js-ui/files/9690373/log.txt)

**Comment by robertsLando** [comment]
Did you tried to re-interview your device first?

**Comment by TutenStain** [comment]
> Did you tried to re-interview your device first?

Yes, I have both re-interviewed the device and have gone through the process of “replacing” it in the UI. All of my many of the same devices exhibits the same behavior. All have worked previously.

**Comment by robertsLando** [comment]
Please attach zwave-js [driver logs](https://zwave-js.github.io/zwave-js-ui/#/troubleshooting/generating-logs?id=generating-log-files)
